package.json

```diff
@@ -54,7 +54,7 @@
     "single-scripture-rcl": "3.4.4",
     "tailwindcss": "^2.0.4",
     "tc-ui-toolkit": "5.3.3",
-    "translation-helps-rcl": "3.4.2",
+    "translation-helps-rcl": "3.4.3-beta",
```

Comment:
Good work cleaning up the code and sorting through the hazards!

Just waiting for you to publish and update translation-helps-rcl